README.md

```diff
@@ -11,8 +11,8 @@ const $ = require('sanctuary-def');
 ```
 
 The next step is to define an environment. An environment is a list of
-[types](#types) such as `$.Number` and `$.String`. `$.env` is a default
-environment, which may be augmented with custom types:
+[types](#types) such as `$.Number` and `$.String`. [`$.env`](#env) is a
+default environment, which may be augmented with custom types:
```

Comment:
I like that but would add further clarity. Most users of sanctuary-def will want to use the technique demonstrated in the example to define environments that extend `$.env`. How's this:

_The next step is to define an environment. An environment is a list of types such as `$.Number` and `$.String`. An environment containing all the builtin JavaScript types can be found at `$.env`. It can be used as the basis for environments that include both the builtin types and custom types:_